MaxText/examples/install_tunix_vllm_requirement.sh

```diff
@@ -0,0 +1,56 @@
+# !/bin/bash
+
+# Copyright 2023–2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script installs the dependencies for running GRPO with MaxText+Tunix+vLLM on TPUs
+
+set -e
+set -x
+
+python -m ensurepip --default-pip
+
+cd ~/maxtext
```

Comment:
lets assume that the user runs from the root maxtext directory, so this probably isn't needed

Reply:
sure. done